### PR TITLE
Make Project Compile Again 

### PR DESCRIPTION
**Pull request by wahyusumartha**
Hi, 

First of all, thanks for making this sdk,
 I was doing some research with Chat SDK that i can use for  my fun project and finally tried Qiscus, 

I was trying to playing around with this project, however i could not compile the project from the first clone, that might be cause by missing file from `Qiscus` Target

Tried on different branch `master` and `develop`, and both has the same issue 

here are my step:
`git clone ...`
`pod install `

FYI I am using XCode 9.4.1 

Basically this pull request is importing missing file to the project so that it could be compiled  

Thank You

### CODE REVIEW

**Comment by asharijuang** [comment]
hi mas @wahyusumartha , thanks for your PR and we really appreciate. 
our documentation in the [website] (https://www.qiscus.com/documentation) we suggest to use other repo for [example](https://github.com/qiscus/qiscus-sdk-ios-sample-v2). This example or project is not represent our SDK, next release we'll remove it. 

I'm happy if you want to try [example](https://github.com/qiscus/qiscus-sdk-ios-sample-v2)

**Comment by wahyusumartha** [comment]
Hi @asharijuang , Thanks
another question, isn't this project is the source of the pod library that is included in that example project that you pointed out before ?

**Comment by asharijuang** [comment]
of cource this project is source of the pod lib, for early version we use example(this) to represent our features.
But other team create awsome other example, a little bit complex and more useful.

**Comment by wahyusumartha** [comment]
I see, thanks a lot for your clarification, 
apologise for all the hassle,

**Comment by asharijuang** [comment]
No problem. i hope our sdk can solve your awesome project. 👍
